### DISCUSSION TITLE
中央通識修課經驗-國會改革與國家發展

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1781] [General]
國會改革與國家發展-劉有恆

這堂課全程線上上課，是用預先錄好的影片上課，所以可以自由選擇上課時間，整學期只有一次期末考跟一個期末報告，期末考其實是像作業一樣可以帶回家寫，答案也都在ppt裡，很好滿分。期末報告則是要找到一個主題然後用上課教的方法去分析這個主題，比較麻煩一點但還在容許範圍內，最終成績為88，有點普通，但如果是想自由分配讀書時間的話很適合!

**Comment by B1nn9** [comment]
這堂課的優點不僅是可以自主分配時間來上課，老師所準備的線上教材也是一點都不馬虎，很多平時對美國國會沒有感受到的細節都有在教材中補充說明，老師也都很歡迎學生跟他私下聯絡討論，是非常有趣的一門課。

**Comment by YANRONG0123** [comment]
我認為這門課也是很棒，只是很少看到有人提到這門課，當初修的時候是開在四34，因為非同步線上授課，所以可以盡量睡過去沒關係，且只要交考卷或報告前一週再開始準備就好，想學知識的人也會有好收穫，因為教材都是逐字稿。
